static/docs/user-guide/experiments/tags.md

```diff
@@ -0,0 +1,115 @@
+# How to Manage Experiments by Tags
+
+You can mark (or label) each experiment using Git tags.
+
+<p align="center">
+<img src="/static/img/user-guide/experiments/tags.png" />
+</p>
+
+Using tags to mark experiments is most suitable when you can easily create a new
+experiment based on the last one. For example when you are trying different
+values for only one aspect of the experiment, say feature extraction.
+
+## Examples
+
+Examples of managing experiments by tags can be seen on these sections of the
+Get Started tutorial:
+
+- [Metrics](/doc/get-started/metrics)
+- [Experiments](/doc/get-started/experiments)
+- [Compare Experiments](/doc/get-started/compare-experiments)
+- [Get Older Files](/doc/get-started/older-versions)
+
+They can also be tried interactively on Katacoda:
+
+- [Experiments and Metrics](https://katacoda.com/dvc/courses/get-started/experiments)
+- [Data Versioning](https://katacoda.com/dvc/courses/get-started/versioning)
+
+## How it works
+
+### Commit and tag
+
+At the end of each experiment we commit everything to Git and add a tag, like
+this:
+
+```dvc
+$ git add .
+$ git commit -m 'Baseline experiment'
+$ dvc commit
+$ git tag baseline-experiment
+```
+
+Do some changes to run an experiment with bigrams and commit it:
+
+```dvc
+$ git commit -am 'Evaluate bigrams model'
+$ dvc commit
+$ git tag bigrams-experiment -am 'Bigrams experiment'
+```
+
+> The command `dvc commit` might not be strictly required, since `dvc repro`
+> usually commits everything, but we are using it just in case there is still
+> some data that is not committed yet.
+
+### Compare the metrics
+
+To find out which experiment has the best performance (the best metrics) we use
+the command `dvc metrics show` with the option `-T, --all-tags`:
+
+```dvc
+$ dvc metrics show -T
+
+baseline-experiment:
+    auc.metric: 0.588426
+bigrams-experiment:
+    auc.metric: 0.602818
+```
+
+### Check out a previous experiment
+
+Let's list first all the tags:
+
+```dvc
+$ git tag
+$ git log --oneline
```

Comment:
show output here?

Reply:
I don't think it will help to improve the explanation in this case.

Reply:
I would improve for me as I read it for the first time. Reading it for the second time - I don't even understand why do we need `git log --oneline` everywhere. You are not saying a word. I meant btw, an output of the `git tag`:

```dvc
$ git tag
baseline
bnigrams
```

it would be easier to read and it is what the previous paragraph is about.

Reply:
Both `git tag` and `git log --oneline` are ways to check what tags are available. Different from `git tag`, `git log --oneline` also shows the position of the tags in the history of commits and their order, which one is first and which one is last.

Reply:
This explanation makes it 10x better! You see, you can do it :) And I'm not kidding - I didn't know the difference between those related to the tags.

I still have some questions though - would log include all commits? If you have a lot of commits (very regular thing to have in real project) tags be lost.

Is there an option for the `git tag` command to sort output?

I would still prefer to simplify it tbh. And put some output - it will make way easier to understand and read. At least you have one reader who is saying that it would be easier to read. Don't rely on your opinion here.